README.md

```diff
@@ -7388,105 +7387,128 @@ See [SQLCipher 4.0.0 Release](https://www.zetetic.net/blog/2018/11/30/sqlcipher-
 
 **You can change the passphrase** of an already encrypted database.
 
-The technique to do so is not the same, depending on whether you are using a [database queue](#database-queues) or a [database pool](#database-pools).
-
-For database queues, first open the database with the old passphrase, and then apply the new passphrase:
+When you use a [database queue](#database-queues), open the database with the old passphrase, and then apply the new passphrase:
 
 ```swift
-// 1. Open the database queue
-var configuration = Configuration()
-configuration.prepareDatabase = { db in
-    try db.usePassphrase("oldSecret")
-}
-let dbQueue = try DatabaseQueue(path: "...", configuration: configuration)
-
-// 2. Later on, change the passphrase
 try dbQueue.write { db in
     try db.changePassphrase("newSecret")
 }
+```
 
-// 3. Keep using the database queue with the new passphrase
-try dbQueue.write { ... }
-try dbQueue.read { ... }
+When you use a [database pool](#database-pools), make sure that no concurrent read can happen during the passphrase change with the `barrierWriteWithoutTransaction` method, and that all future reads open a new database connection with the `invalidateReadOnlyConnections` method:
```

Comment:
🤓 language nerd here, but the "and that all future reads" is pretty far from the "make sure that" to which it applies.

Maybe this is a little clearer:

When you use a [database pool](#database-pools), make sure that no concurrent read can happen by changing the passphrase within the `barrierWriteWithoutTransaction` block. You must also ensure all future reads open a new database connection by calling the `invalidateReadOnlyConnections` method:

Reply:
Thanks Michael, your way to say it is so much better 👍  cc7d19d92567c6c71ba80ba474db7df5fc72afe6